src/components/PolicyTopologyPage.tsx

```diff
@@ -73,6 +72,9 @@ const fetchConfig = async () => {
   }
 };
 
+export const kindToAbbr = (kind: string) => {
```

Comment:
import { kindToAbbr } from '../utils/modelUtils';

Reply:
i removed the function from that file since it was now the only thing in it with the deletion of the DNS stuff but i can move it back

Reply:
nah, let's leave it here for now. if we have a need for re-use again, can make another util.